### PR TITLE
pyth2 working 

### PR DESCRIPTION
**Pull request by nope-finance**

### CODE REVIEW

File: token-lending/program/src/pyth.rs [hunk 1]
```diff
@@ -67,29 +67,34 @@ pub struct PriceComp {
...
-    pub num: u32,         // Number of component prices.
-    pub unused: u32,
-    pub curr_slot: u64,  // Currently accumulating price slot.
-    pub valid_slot: u64, // Valid slot-time of agg price.
-    pub prod: AccKey,
-    pub next: AccKey,
-    pub agg_pub: AccKey,
-    pub agg: PriceInfo,
-    pub comp: [PriceComp; 16],
+  pub magic      : u32,        // pyth magic number
```
Comment (0xrooters) [inline token-lending/program/src/pyth.rs:75]: Which plugin are you using for this? Imma cop

**Comment by 0xrooters** [comment]
Nice, so no API changes? Pretty simple.

File: token-lending/program/src/pyth.rs (hunk 1 above)
Comment (nope-finance) [reply token-lending/program/src/pyth.rs:75]: plugin?

**Comment by nope-finance** [comment]
only breaking change is version mismatch. everything else is just adding new fields i think yeh

File: token-lending/program/src/pyth.rs (hunk 1 above)
Comment (nope-finance) [reply token-lending/program/src/pyth.rs:75]: apparently doing this causes rustfmt to fail anyway LOL

**Pull request closed without merging**